dds/DCPS/DataReaderImpl.h

```diff
@@ -417,11 +417,13 @@ class OpenDDS_Dcps_Export DataReaderImpl
 
   bool is_bit() const;
 
-  /** This method is used for a precondition check of delete_datareader.
+  /**
+   * This method is used for a precondition check of delete_datareader.
    *
-   * @returns the number of outstanding zero-copy samples loaned out.
+   * @retval true We have zero-copy samples loaned out
+   * @retval false We have no zero-copy samples loaned out
    */
-  virtual int num_zero_copies();
+  virtual bool has_zero_copies();
```

Comment:
I know you didn't change this, but as I reviewed it I noticed that there's no reason for this to be virtual.